components/contribution-flow/index.js

```diff
@@ -624,13 +660,14 @@ class ContributionFlow extends React.Component {
     const { error, isSubmitted, isSubmitting, stepDetails, stepSummary, stepProfile, stepPayment } = this.state;
     const isCrypto = paymentMethod === 'crypto';
     const currency = isCrypto ? stepDetails.currency.value : tier?.amount.currency || collective.currency;
+    const isLoading = isCrypto ? isSubmitting : isSubmitted || isSubmitting;
 
     return (
       <Steps
         steps={this.getSteps()}
         currentStepName={this.props.step}
         onStepChange={this.onStepChange}
-        onComplete={this.submitOrder}
+        onComplete={isCrypto && isSubmitted ? this.cryptoOrderCompleted : this.submitOrder}
```

Comment:
why not handle this from `submitOrder`?

Reply:
So the completion of the crypto flow means we are just showing the success page. But our contribution flows default behavior upon completion means creating the order and then showing the success page basically. Now when we put this `cryptoOrderCompleted` logic within `submitOrder` function it will be sidestepping the default behavior of the `submitOrder` function. In fact, when the `submitOrder` function here is called in the crypto flow, it's actually never doing any submission, the submission is already done at, 

https://github.com/opencollective/opencollective-frontend/blob/7bae43cfc4ae1daef0ce3bd8bda8ffc1f65078a0/components/contribution-flow/index.js#L411

So moreover we are calling the `submitOrder` function twice, and the second time it will never do any `submitOrder` at all, which is misleading. This is why it was done this way. Let me know if you disagree. 🙂